frontend/services/entities/mdm.ts

```diff
@@ -192,19 +193,17 @@ const mdmService = {
   // disk encryption too.
   updateAppleMdmSettings: (enableDiskEncryption: boolean, teamId?: number) => {
     const {
-      MDM_UPDATE_APPLE_SETTINGS: teamsEndpoint,
+      UPDATE_DISK_ENCRYPTION: teamsEndpoint,
       CONFIG: noTeamsEndpoint,
     } = endpoints;
     if (teamId === 0) {
       return sendRequest("PATCH", noTeamsEndpoint, {
         mdm: {
-          // TODO: API INTEGRATION: remove macos_settings when API change is merged in.
-          macos_settings: { enable_disk_encryption: enableDiskEncryption },
-          // enable_disk_encryption: enableDiskEncryption,
+          enable_disk_encryption: enableDiskEncryption,
```

Comment:
Is this supported on the backend, or does this tweak need to be built so we don't lose support for macOS?

Reply:
Above should answer this question as well

Reply:
(yes, it's supported 🙂)